### ISSUE TITLE
Training set duplication problem

### ISSUE THREAD

**Issue by hxy-daniel** [Messi-Q/GNNSCVulDetector#3]
Hello, I have studied your paper and code, and want to ask some questions:
1. Why is there a lot of repeated data in this training set, and so is the validation set? Why do you need to do this?
2. Why are the feature results different after these repeated training sets are processed by your network ?(in the reentrancy_train_feature_with_rnn_cell.txt file)
